app/assets/javascripts/discourse/app/components/discovery/topics.hbs

```diff
@@ -141,7 +141,7 @@
     {{/if}}
   {{/if}}
 
-  <span>
+  <span class="after-topic-list-plugin-outlet-wrapper">
```

Comment:
All these wrapping `span`s are legacy code and invalid html (a div in a span? what is this madness! 😄) and we're slowly removing them, one by one to avoid breaking stuff. If we actually do want a wrapper in this case I'd recommend changing it to a `div`

```suggestion
  <span class="after-topic-list-plugin-outlet-wrapper">
```

Reply:
@CvX I don't want this here... but since it is here I need to select it... The good news is that having it named will make searching code that relies on it ez pz

Reply:
But I don't feel comfortable removing the wrapper and breaking stuff rn 😬